TOOLS.md

```diff
@@ -0,0 +1,47 @@
+# Tools
```

Comment:
It seems that the new table lost entries that are not managed by manifest from the previous, such as nextest, valgrind.

Reply:
I've done a comparison now, and it is only nextest and valgrind from README.md that dont appear in TOOLS.md. 

- nextest is easy - we can create a base manifest for it, to get the bins at https://github.com/nextest-rs/nextest/releases/tag/cargo-nextest-0.9.70

- That leaves `valgrind`, which is special as it isnt on GitHub.

Not too hard to have a special case for `valgrind`.

But, I have also noticed that `cargo-watch` & `watchexec-cli` is in https://github.com/taiki-e/install-action/blob/689459d9ffef015a7fbaef7f3b6b9f053f80a64d/tools/ci/tool-list.sh#L135-L138 , but these are not in README.md .  Any others that you know about ?

cargo-watch can also be a base manifest, getting bins from https://github.com/watchexec/cargo-watch/releases/tag/v8.5.2

watchexec-cli might be the bins at https://github.com/watchexec/watchexec/releases/tag/v2.1.1 - need to double check that one.

I see quite a few other system tools that are installed by https://github.com/taiki-e/install-action/blob/main/main.sh .  Should we be mentioning them also?  `jq`, `curl`, `tar` also ?

They would be more special cases, which is fine.

- jq could be a manifest - see https://github.com/jqlang/jq/releases/tag/jq-1.7.1
- https://github.com/curl/curl/releases/tag/curl-8_7_1 doesnt contain binaries
- Maybe we could be replacing `tar` with https://github.com/vadorovsky/packy/releases/tag/v0.0.3 , if it released more arch binaries, and it uploaded untar'd bins to avoid chicken and egg problem.

Reply:
> But, I have also noticed that `cargo-watch` & `watchexec-cli` is in

tools/ci/tool-list.sh is only for testing. tools/publish.sh is for "Supported tools".

https://github.com/taiki-e/install-action/blob/689459d9ffef015a7fbaef7f3b6b9f053f80a64d/tools/publish.sh#L105-L106

> I see quite a few other system tools that are installed by https://github.com/taiki-e/install-action/blob/main/main.sh . Should we be mentioning them also? jq, curl, tar also ?

Let's ignore them at this PR, as they are for incomplete runners and are not usually installed by us.